projects/experimental/components/cell/cell.component.ts

```diff
@@ -0,0 +1,33 @@
+import {ChangeDetectionStrategy, Component, Inject, Input} from '@angular/core';
+import {TUI_PLATFORM, TuiPlatform} from '@taiga-ui/cdk';
+import {TuiSizeL, TuiSizeS} from '@taiga-ui/core';
+import {PolymorpheusContent} from '@tinkoff/ng-polymorpheus';
+
+@Component({
+    selector: 'tui-cell',
+    templateUrl: './cell.template.html',
+    styleUrls: ['./cell.styles.less'],
+    changeDetection: ChangeDetectionStrategy.OnPush,
+    host: {
+        '[attr.data-size]': 'size',
+        '[attr.data-platform]': 'platform',
+    },
+})
+export class TuiCellComponent {
+    @Input()
+    title: PolymorpheusContent = '';
```

Comment:
It seems like sort of layout component, should we use content projection with `tuiSlot` for this type of component to unify our API? should we make some rules about this?

what do you think @waterplea @splincode @nsbarsukov ?

Reply:
fixed ✅

Reply:
fixed ✅